src/components/CellLineTable/index.tsx

```diff
@@ -77,12 +99,33 @@ const CellLineTable = ({
     const interactiveColumns = columns.map((column: any) => {
         // the two clickable columns are the order cell line and
         // CoA column. They do not have the hover effect and
-        // should not take you to the cell line page
+        // should not take you to the cell line page, and are not
+        // sortable.
         if (column.className?.includes("action-column")) {
             return column;
         }
+        const isSortedColumn = sortedColumn.key === column.key;
+        const sortIcon =
```

Comment:
non-blocking: from my personal user experience, adding tooltips like `Sort A-Z` or `Ascending`/`Descending` to the sort icon would be very helpful.